build/Targets/SettingsSdk.props

```diff
@@ -4,7 +4,7 @@
 
   <PropertyGroup>
     <RoslynSdkProject>true</RoslynSdkProject>
-    <EnableDefaultItems>false</EnableDefaultItems>
+    <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
```

Comment:
None globs are, in my opinion, a mistake we made in the SDK. Changing it now is a breaking change - we've had some discussions about doing it in a non-breaking way but avoiding it here entirely.